x-pack/plugins/uptime/public/components/common/step_detail_link.tsx

```diff
@@ -23,10 +22,14 @@ export const StepDetailLink: FC<StepDetailLinkProps> = ({ children, checkGroupId
   const to = `/journey/${checkGroupId}/step/${stepIndex}`;
 
   return (
-    <EuiLink>
-      <Link data-test-subj={`step-detail-link`} to={to}>
-        {children}
-      </Link>
-    </EuiLink>
+    <ReactRouterEuiButtonEmpty
+      data-test-subj={`step-detail-link`}
+      to={to}
+      size="s"
+      iconType="apmTrace"
```

Comment:
```suggestion
      iconType="apmTrace"
```

I think we discussed replacing this in order to not confuse users who might understand this icon as APM traces. @katrin-freihofner what do you think? Should we replace with a simple `arrowRight` like the Step details below? Another option is to convert this to `EuiButton` for a clearer primary call to action?